janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLDistributedStoreManagerTest.java

```diff
@@ -54,6 +55,7 @@ public void tearDown() throws BackendException {
     @Override
     @Test
     @FeatureFlag(feature = JanusGraphFeature.OrderedScan)
+    @Disabled(value = "Test can't run inside of a container")
```

Comment:
I think we may need to add an issue to either fix this test to be able to run in GitHub Actions or rewrite this test. So that we could enable it back later. As for now I'm fine disabling it

Reply:
For me, we could also remove the test. I don‘t see any value of the current test.

Reply:
Let's just open an issue for this test and then either fix it or remove as a separate PR. Don't remove it in this PR

Reply:
https://github.com/JanusGraph/janusgraph/issues/2303